sql-plugin/src/main/scala/com/nvidia/spark/rapids/lore/GpuLore.scala

```diff
@@ -90,7 +90,7 @@ object GpuLore {
 
   def dumpObject[T: ClassTag](obj: T, path: Path, hadoopConf: Configuration): Unit = {
     withResource(path.getFileSystem(hadoopConf)) { fs =>
```

Comment:
Not part of this PR, but typically we want to leave the underlying FileSystem.get calls memoized to make the subsequent ones cheaper.  Thus the FileSystem object should not be closed via withResource